databricks/koalas/frame.py

```diff
@@ -9992,6 +9993,63 @@ def eval_func(pdf):
             # Returns a frame
             return result
 
+    def explode(self, column):
+        """
+        Transform each element of a list-like to a row, replicating index values.
+
+        Parameters
+        ----------
+        column : str or tuple
+            Column to explode.
+
+        Returns
+        -------
+        DataFrame
+            Exploded lists to rows of the subset columns;
+            index will be duplicated for these rows.
+
+        See Also
+        --------
+        DataFrame.unstack : Pivot a level of the (necessarily hierarchical)
+            index labels.
+        DataFrame.melt : Unpivot a DataFrame from wide format to long format.
+
+        Examples
+        --------
+        >>> df = ks.DataFrame({'A': [[1, 2, 3], [], [3, 4]], 'B': 1})
+        >>> df
+                   A  B
+        0  [1, 2, 3]  1
+        1         []  1
+        2     [3, 4]  1
+
+        >>> df.explode('A')
+             A  B
+        0  1.0  1
+        0  2.0  1
+        0  3.0  1
+        1  NaN  1
+        2  3.0  1
+        2  4.0  1
+        """
+        if isinstance(column, tuple) or is_scalar(column):
+            spark_type = self[column].spark_type
+            if not isinstance(spark_type, ArrayType):
+                return self
+            column = name_like_string(column)
+        else:
+            raise ValueError("column must be a scalar")
+
+        sdf = self._sdf
+        sdf = sdf.withColumn(column, F.explode_outer(sdf[column]))
+        internal = InternalFrame(
+            spark_frame=sdf,
+            index_map=self._internal.index_map,
+            column_labels=self._internal.column_labels,
+            column_label_names=self._internal.column_label_names,
+        )
```

Comment:
Can you try to copy instead of creating new one? `self._internal.copy(...)`

Reply:
@HyukjinKwon Fixed it, thanks! :)